tests/scalers/prometheus-value-metric-type.test.ts

```diff
@@ -0,0 +1,165 @@
+import * as async from 'async'
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import test from 'ava'
+import {waitForRollout} from "./helpers";
+
+const testNamespace = 'prometheus-test'
+const prometheusNamespace = 'monitoring'
```

Comment:
We are running the e2e tests in parallel, so we should use unique namespace for each test, so could you please rename this? And the same for the prometheus installation, could you please check the latest version of prometheus e2e test? https://github.com/kedacore/keda/blob/main/tests/scalers/prometheus.test.ts#L5

Reply:
you're right @zroubalik .
Please use a unique name for this namespace and use `PrometheusServer` helper to create all needed resources. We discovered some race conditions running multiple e2e test related with Prometheus in parallel if we don't manage each server split